#request

---
#response
nav_msgs/OccupancyGrid map_msg
